internal/rbd/rbd_attach.go

```diff
@@ -239,6 +241,9 @@ func appendDeviceTypeAndOptions(cmdArgs []string, isNbd, isThick bool, userOptio
 		if !strings.Contains(userOptions, setNbdReattach) {
 			cmdArgs = append(cmdArgs, "--options", fmt.Sprintf("%s=%d", setNbdReattach, defaultNbdReAttachTimeout))
 		}
+		if !strings.Contains(userOptions, setLogFile) {
```

Comment:
IMO we should not default this one. not useful for everyone. but as you mentioned this can be configured in the storageclass hope that is enough.

Reply:
AFAIK, there is no way to set the log-dir for rbd clients, there is only a way to set log-file by the complete path.
I want to have a consistent naming for rbd-nbd log file like: rbd-nbd.log, unlike ceph-client.$id.log

If you don't like it feel free to show up :+1: I can remove it.

Reply:
No, what I meant was not to set default logging if someone does not specify it in the storageclass.
one more question, If multiple clients write to the same file how do identify which pod/client the log belongs to?

Reply:
> No, what I meant was not to set default logging if someone does not specify it in the storageclass.

So, I get you right. We don't need a standard logfile name like: rbd-nbd.log ? You mean to use the ceph-client.admin.log name'ing?

> one more question, If multiple clients write to the same file how do identify which pod/client the log belongs to?

This is something I'm working on currently, ideally rbd-nbd should prefix the device number (like nbd0, nbd1 .. etc) which is not the case currently. But in the future expect it logs with the device index as prefix.

Thanks!